lib/Collection/QueryResultCollection.php

```diff
@@ -52,21 +47,19 @@ class QueryResultCollection extends AbstractLazyCollection
     private $primarySelector = null;
 
     /**
-     * @param QueryResultInterface $result
-     * @param EventDispatcherInterface $eventDispatcher
-     * @param string $locale
-     * @param array $options
-     * @param null|string $primarySelector
+     * @var DocumentManagerInterface
      */
+    private $manager;
+
     public function __construct(
```

Comment:
I think @chirimoya recently said that he'd like to have the phpdoc also for constructor calls.